prov/efa/src/rxr/rxr_pkt_type_data.c

```diff
@@ -207,7 +207,10 @@ ssize_t rxr_pkt_send_data_desc(struct rxr_ep *ep,
 		i++;
 	}
 
-	pkt_entry->send = malloc(sizeof(struct rxr_pkt_sendv));
+	pkt_entry->send = ofi_buf_alloc(ep->pkt_sendv_pool);
+	if (!pkt_entry->send)
```

Comment:
Do we need to release packet entry here?

Also, add a warning here.

Reply:
No, the caller `rxr_pkt_post_data` will release it.

Reply:
Thanks!